client/lib/domains/registration/availability-messages.js

```diff
@@ -254,7 +254,7 @@ function getAvailabilityNotice( domain, error, errorData ) {
 				);
 			} else {
 				message = translate(
-					'Domain cannot be mapped to a WordPress.com blog because of blacklisted term.'
+					'Domain cannot be mapped to a WordPress.com blog because of disallowed term.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.